res/raw-sw600dp/linphonerc_default

```diff
@@ -35,4 +35,21 @@ external_domains=Sorenson.com,ZVRS.com,CAAG.com,PurpleVRS.com,GlobalVRS.com,Conv
 
 [rtp]
 rtcp_xr_enabled=0
+ 
+[video_codec_0]
+mime=H264
+rate=90000
+enabled=1
+recv_fmtp=profile-level-id=42801F
```

Comment:
I'm not sure I agree with that profile-level-id=42801f

The 42h means baseline profile which is fine, and the 80 means it follows certain bit constraints, but the 3.1 means baseline profile with macroblock updates able to support 1280x720@30fps... Are we saying we want to support up to 720p video here?

Also, shouldn't we be setting the packetization mode for both 0 and 1?

recv_fmtp=packetization-mode=1